templates/vagrantfiles/Vagrantfile.2master_2replica_2client

```diff
@@ -0,0 +1,81 @@
+# -*- mode: ruby -*-
```

Comment:
I would give the file a different name, for instance `Vagrantfile.ipa_ipa_trust` to make sure it's not used for cases where a test needs 4 servers and 2 clients.